packages/ui/src/helpers/accountSettings/validators/password.ts

```diff
@@ -0,0 +1,84 @@
+import { Amplify } from 'aws-amplify';
+import { hasSpecialChars } from '../../../helpers/authenticator';
+import { FieldValidator, PasswordSettings } from '../../../types';
+
+/** Gets password setting from Amplify configuration */
+export const getPasswordSettings = () => {
+  // need to cast to any because `Amplify.configure()` isn't typed properly
+  const config = Amplify.configure() as any;
+  return config?.aws_cognito_password_protection_settings as PasswordSettings;
+};
+
+export const getMinLengthValidator = (minLength: number): FieldValidator => ({
+  validationMode: 'onTouched',
+  validate: (field) => field.length >= minLength,
```

Comment:
nit: I wonder if calling it `isValid` more clearly indicates that it returns a boolean?

Reply:
That's fair. I personally think `isValid` sounds like a boolean than a function that returns boolean, so I'm hesitant on the change.

Wonder what others think too

Reply:
Resolving, changed to `validator`.